src/analysis/runtime-common-components.js

```diff
@@ -4,8 +4,8 @@ import _ from 'lodash/fp';
 import { Fragment, useEffect, useState } from 'react';
 import { b, div, h, input, label } from 'react-hyperscript-helpers';
 import { getConvertedRuntimeStatus, usableStatuses } from 'src/analysis/utils/runtime-utils';
+import { cookiesAcceptedKey } from 'src/auth/accept-cookies';
```

Comment:
flipped dependency direction to break a circular reference unrelated to the main code changes in the PR.